webapp/cas-server-webapp-tomcat-starter/src/main/java/org/apereo/cas/CasStarterBanner.java

```diff
@@ -0,0 +1,23 @@
+package org.apereo.cas;
+
+import org.apereo.cas.util.spring.boot.AbstractCasBanner;
+
+import org.apache.catalina.util.ServerInfo;
+import org.springframework.core.env.Environment;
+
+import java.util.Formatter;
+
+/**
+ * This is {@link CasStarterBanner}.
+ *
+ * @author Hal Deadman
+ * @since 6.5.0
+ */
+public class CasStarterBanner extends AbstractCasBanner {
```

Comment:
You likely should add a test case to make sure the formatting and the injection works OK.

Reply:
I copied the tests from tomcat webapp and ran them on this starter banner, not sure if there is something else I should test besides the banner works and app context loads (if that is what CasWebApplicationServletInitializerTests is testing).